### ISSUE TITLE
Where is this mod going?

### ISSUE THREAD

**Issue by WolfieWaffle** [BrassGoggledCoders/SteamCraft2#342]
I really like this mod, but I think it needs more of a plan/direction. Right now it's very similar to Flaxbeard's Steam Power, which isn't necessarily a bad thing, but I think it needs a more specific idea. The only way I can really classify this mod right now is that it's based off steam power.

I think the mod needs some kind of "revitalization" or something, so we have a more clear direction to go in. And I know I probably don't have much authority on the mod, but this is just my ideas of what to do.

I would like to see more compatibility with other mods and focusing more on things that other mods don't add. Like plenty of mods have and ore doubling system, but not many have a ray gun. So I think the mod should focus on things like that, making sure they work really well. I'd really like to see a lot of compatibility with Immersive engineering, maybe adding some new wires or something.

But of course not just mod comparability. I just get carried away with that because I like making mod packs :P. Anyways I think some high-voltage stuff would be cool. I think if we added stuff in the line of that purple crystal I forgot the name of, the ray gun, and phosphate, it could differentiate from other steam mods. I don't really know honestly, I'm just trying to get ideas. Maybe the steam powered ladder mentioned in #337 , or glowing phosphate paint and giant steam guns and stuff.

Anyways, I basically think we should juts make a kind of plan for where to take the mod, because right now it seems like it doesn't have too much direction. I would like to see more stuff with cool gadgets and high-powered machinery, but that's just me. So what does everyone else think?

**Comment by warlordjones** [comment]
I agree! My todo list has plenty of unique features *taps nose* But for now, I need to work on the deeps and getting that stable if at all possible, since that's our first truly unique feature. That and the next major update is hopefully going to be a machinery update in which I flesh out the steam and RF power systems. The injector/extractor WIP is just one example of that. Check out the machinery masterlist (#305) too for more plans. I've effectively got a rough roadmap of where I want to go, check the milestones page.

**Comment by WolfieWaffle** [comment]
Yeah, the deeps would be good, maybe a certain ore could be only there. I'd like to work on the deeps, but I could only do ideas and stuff because I can't do terrain generation and I never got the eclipse thing to work. But it could be a cool place to explore, or make a secret base or something. I looked at the machinery masterlist, and it looks pretty cool. It seems like mostly machinery stuff, like ore processing and power/steam manipulation. So I think it would be good to get that stuff, but then make some unique machines or systems that are based off that. I don't know how good you are with multiblocks, but some Immersive Engineering type machines could be cool, but more steam-y and lightning-y and glowy (uranium maybe?). And more things related to ertherium.

So yeah, I think the machinery should come first, and then work on adding unique features. I think the strongest unique features of this mod are the etherium, the lightning rod, the ray gun, and the steam suit. I think if after the machinery we added some things focusing on those things, it would be awesome. More "big" stuff, like the lightning rod, and "weird" stuff, like etherium, and wasn't there a white spider? And maybe phosphate. (idk what my thing is with the phosphate, I just love its color and it makes me imagine magical blue particles and glowyness).

Anyways that was longer than I expected, but I think the next updates are going to be cool :)

**Comment by SkySom** [comment]
Yeah there's a bit more mod compat coming (You can see the few issues I created) Most are for the armor set (Which only a few other mods have modular armor)

**Comment by HCSarise** [comment]
What about a golems style thing (I know Thaumcraft already does this, but style it for your mod). Think Cogsworth in Fallout4. Little golems/robots you can get to do stuff for you, very similar to thaumcraft. As far as I know there's not really a techy-feeling version of TC golems.

**Comment by SkySom** [comment]
Buildcraft's Robots would be the closest Techy version of Golems.

**Comment by warlordjones** [comment]
Automatons? Vague ideas list has a few ideas for that...I'll make a compilation issue for them. Won't be coming for a long while, mind.

**Comment by HCSarise** [comment]
In the theme of gadgetry, how about some goggles where you can select different lenses/overlays (something like this https://img1.etsystatic.com/006/1/7002697/il_570xN.469704629_2pth.jpg) such as perhaps a hp/hud overlay, maybe an ore sensing overlay, Thaumcraft integration, and the like?

**Comment by warlordjones** [comment]
Sounds cool, but pretty much the sort of thing I plan for brass armour helmet modules. Some of those are certainly good ideas for modules though!

**Comment by SkySom** [comment]
Yeah... Still need to finish the rewrite of the modules... Welp will put that on the list for when I finally start the 1.8.9 port for SteamCraft.

**Comment by SkySom** [comment]
And the night vision is already implemented.

**Comment by warlordjones** [comment]
1.8? *shudders* :P 

Did I already do night vision? Oh XD

**Comment by SkySom** [comment]
Yes I've used it multiple times. And pfft. Boilerplate/MoarCarts (minus some rendering when I should have done tonight) are both ported.

**Comment by SkySom** [comment]
Though some of Boilerplate is questionable as I had no idea how it worked. So I just kinda made it stop throwing errors and figured I'd fix it when I found out what was using it.

**Comment by warlordjones** [comment]
...oh. glad to have you onboard :P

**Comment by SkySom** [comment]
yeah probably won't get around to SteamCraft too soon. (Figured I'd get MoarCarts running well, and then maybe do BrassUtils). But it's on my radar now that 1.8.9 is alive and well within the community

**Comment by TheDoctorSoda** [comment]
I've completely switched to 1.8.9. It's really not that different once you
get used to it. I think you guys should hop on the 1.8.9 train too.

**Comment by SkySom** [comment]
Like I said. I've hopped on it in so far as starting to port MoarCarts. Just not sure when I'll have time to get to SteamCraft.
